tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/charts/Course_Information_f909e0.yaml

```diff
@@ -47,31 +42,56 @@ params:
     column: median_course_grade
     operator: <
     targetValue: 50
+  - colorScheme: '#ACE1C4'
+    column: Active Within Last 7 Days
+    operator: '>'
+    targetValue: 0
   extra_form_data: {}
   groupby:
   - org
   - course_key
   - course_name
   - course_run
   metrics:
-  - active_within
+  - aggregate: AVG
+    column:
+      advanced_data_type: null
+      certification_details: null
+      certified_by: null
+      column_name: active_within_last_7_days
+      description: null
+      expression: null
+      filterable: true
+      groupby: true
+      id: 683
+      is_certified: false
+      is_dttm: false
+      python_date_format: null
+      type: UInt64
+      type_generic: 0
+      verbose_name: Active Within Last 7 Days
+      warning_markdown: null
+    datasourceWarning: false
+    expressionType: SIMPLE
+    hasCustomLabel: true
+    label: Active Within Last 7 Days
+    optionName: metric_9ysvoht7z9_7zzqrfsnybt
```

Comment:
Metrics should be added to the datasets, instead of using the charts metrics, (translations)